service/src/batch_prover_service.rs

```diff
@@ -0,0 +1,7 @@
+use prover_scheduler::service::batch_prover_service::launch_batch_prover_service_with_addr;
+
+#[tokio::main]
+async fn main() -> Result<(), Box<dyn std::error::Error>> {
+    let addr = "http://[::1]:8545".to_string();
```

Comment:
Set up the address by environment variable.

Reply:
ok

Reply:
done!